docs/source/activity-plans/scheduling-guide.rst

```diff
@@ -126,7 +126,7 @@ a goal is enabled or disabled.
 
 Scheduling DSL Documentation
 ============================
-Here you will find the full set of features in the `Scheduling EDSL API Documentation <../../scheduling-edsl-api/{{current_version}}/index.html>`__.
+The full set of features can be found in the Scheduling EDSL API Documentation <../../scheduling-edsl-api/{{current_version}}/index.html>__.
```

Comment:
You accidentally dropped the backticks around the link ;-)

Reply:
Darn copy paste doing weird string sanitization